src/main/scala/edu/berkeley/ce/rockslicing/BlockVTK.scala

```diff
@@ -0,0 +1,197 @@
+package edu.berkeley.ce.rockslicing
+
+import breeze.linalg
+import breeze.linalg.{DenseVector, DenseMatrix}
+
+object BlockVTK {
+
+  private val EPSILON = 1.0e-6
+
+  /**
+   * Calculates the rotation matrix to rotate the input plane (specified by its normal)
+   * to the desired orientation (specified by the desired normal)
+   * @param n_current: Current normal of plane
+   * @param n_desired: Desired new normal
+   * @return 3*3 rotation matrix
+   */
+  private def rotationMatrix(n_current: (Double, Double, Double), n_desired: (Double, Double, Double)):
+  DenseMatrix[Double] = {
```

Comment:
You should indent this line, maybe as far as the opening parenthesis on the preceding line.